dashboard/app/models/levels/level_group.rb

```diff
@@ -213,7 +213,7 @@ def self.get_levelgroup_survey_results(script_level, section)
   #           {result: "I like making games, and I also like the lifestyle."},
   #           {}]}]}]
   #
-
+  # TODO(caleybrock): remove once assessments tab is converted to react
   def self.get_survey_results(script, section)
```

Comment:
This function is very similar to the new one (but returns an array instead of a hash). This function is not tested at all, and the plan is to remove it once we entirely switch to the new API/react.